Sources/MessagingPush/MessagingPush+NotificationCenter.swift

```diff
@@ -18,9 +18,18 @@ public extension MessagingPush {
         didReceive response: UNNotificationResponse,
         withCompletionHandler completionHandler: @escaping () -> Void
     ) -> Bool {
+        guard let siteId = customerIO.siteId else {
```

Comment:
been thinking a little about all local state being held relative the `siteId` - I think the most likely case for a `siteId` ever changing is going to be in a scenario where it needs to be rotated for some reason, and doing so will lose any local context / pending work. Don't know that there's a great solution for that, except that we should at least have some recommendations prepared for a future customer as to how to safely rotate their key.

no action items for this, just food for thought